common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2654,12 +2654,12 @@ StScrollBar {
   .notification,
   .unlock-dialog-notification-source {
     padding: 12px 6px;
-    border: 1px solid $_bubble_borders_color;
-    background-color: transparentize($osd_bg_color,0.5);
-    color: $_bubble_fg_color;
+    border: 1px solid transparentize(black, 0.9);
+    background-color: transparentize($shell_popup_menu_background_color, 0.5);
+    color: $fg_color;
     border-radius: 4px;
 
-    &.critical { background-color: transparentize($osd_bg_color,0.1) }
+    &.critical { background-color: transparentize($shell_popup_menu_background_color, 0.1) }
```

Comment:
I think this needs some thinking on the design. Maybe just comment this out for now, and mark with `//FIXME` at the end, until the overall lock screen notification design is set.

Reply:
Commented out as suggested. Might need to use different background and/or foreground color to highlight critical notifications